tee-worker/client-api/parachain-api/prepare-build/interfaces/vc/definitions.ts

```diff
@@ -0,0 +1,73 @@
+export default {
+    types: {
+        VCRequested: {
+            account: "AccountId",
+            mrEnclave: "ShardIdentifier",
+            assertion: "Assertion",
+        },
+        Assertion: {
+            _enum: {
+                A1: "Null",
+                A2: "Bytes",
+                A3: "(Bytes,Bytes,Bytes)",
+                A4: "Bytes",
+                A6: "Null",
+                A7: "Bytes",
+                A8: "Vec<AssertionSupportedNetwork>",
+                A10: "Bytes",
+                A11: "Bytes",
+                A12: "Bytes",
+                A13: "AccountId32",
+                A14: "Null",
+            },
+        },
+        AssertionSupportedNetwork: {
+            _enum: ["Litentry", "Litmus", "LitentryRococo", "Polkadot", "Kusama", "Khala", "Ethereum", "TestNet"],
+        },
+        RequestVCResult: {
+            vc_index: "H256",
+            vc_hash: "H256",
+            vc_payload: "AesOutput",
+        },
+        ErrorDetail: {
+            _enum: {
+                ImportError: "Null",
+                UnauthorizedSigner: "Null",
+                StfError: "(Bytes)",
+                SendStfRequestFailed: "Null",
+                UserShieldingKeyNotFound: "Null",
+                ParseError: "Null",
+                DataProviderError: "(Bytes)",
+                InvalidIdentity: "Null",
+                WrongWeb2Handle: "Null",
+                UnexpectedMessage: "Null",
+                WrongSignatureType: "Null",
+                VerifySubstrateSignatureFailed: "Null",
+                VerifyEvmSignatureFailed: "Null",
+                RecoverEvmAddressFailed: "Null",
+                Web3NetworkOutOfBounds: "Null",
+            },
+        },
+        StfError: {
```

Comment:
This type is not specific to VC, it's just sidechain's rpc type.

Reply:
same as above.